src/ai/defaultai.cc

```diff
@@ -280,9 +280,9 @@ void DefaultAI::think() {
 	// range 13 - 15, this is used for training of AI
```

Comment:
this is a leftover from the original place of the auto speed function which I forgot to correct. Could you correct this too please.  
// This portion of code keeps the speed of game to ensure AI tasks
	// being on time, this is used for training of AI

Reply:
replaced